config/core/rootfs-configs.yaml

```diff
@@ -195,21 +195,32 @@ rootfs_configs:
       - wget
       - xz-utils
 
+  bullseye-kvm-unit-tests:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+      - amd64
+      - arm64
+    extra_packages:
+      - qemu-system
+    extra_packages_remove:
+      - bash
+      - e2fslibs
+      - e2fsprogs
+    script: "scripts/bullseye-kvm-unit-tests.sh"
+
   bullseye-libcamera:
     rootfs_type: debos
     debian_release: bullseye
     arch_list:
       - amd64
       - arm64
       - armhf
-    extra_packages:
-      - libevent-dev
```

Comment:
Accidental changes? This is a different rootfs. Same below.

Reply:
Ugh, this is junk due to having to rebase this for so long - git must've got confused at some point.